.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: [ '1.16.x', '1.17.x' ]
+        go: [ '1.16.x', '1.17.x', '1.20.x' ]
```

Comment:
We should remove 1.16 and 1.17 and keep 1.19 and 1.20. Go officially only supports last 2 releases.

Reply:
Done